app/src/main/res/layout/activity_main.xml

```diff
@@ -9,12 +9,22 @@
     <include layout="@layout/content_main" />
 
     <com.google.android.material.floatingactionbutton.FloatingActionButton
-        android:id="@+id/fab"
+        android:id="@+id/fabCamera"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_gravity="bottom|center"
+        android:layout_gravity="bottom|start"
         android:layout_margin="@dimen/fab_margin"
         app:fabSize="normal"
         app:srcCompat="@drawable/ic_camera_alt_white_24dp" />
 
+    <com.google.android.material.floatingactionbutton.FloatingActionButton
```

Comment:
@dineshkarthi95 Using two FABs on the same screen is not recommended on the [Material Guidelines](https://m3.material.io/components/floating-action-button/guidelines#1eacb352-bd1c-48e1-8e84-853ceb79ec94). Maybe we could replace them with regular buttons on the screen?

Reply:
Yeah sounds cool we can use ImageButtons or something and remove the FAB's

Reply:
@thatfiredev Have updated the PR with Image Buttons :)